addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java

```diff
@@ -122,14 +124,18 @@ private String getDetails(String key, Map<String, Set<String>> info) {
         }
         StringBuilder sb = new StringBuilder();
         for (String item : info.get(key)) {
+            if (key.equals(Result.CVE)) {
+                alertTags.put(item, "");
+            }
```

Comment:
This should be done explicitly (e.g. in `buildAlert`) instead of relying on side effects.